packages/eslint-plugin/src/rules/member-ordering.ts

```diff
@@ -329,6 +334,8 @@ const functionExpressions = [
  * @param node the node to be evaluated.
  */
 function getNodeType(node: Member): MemberKind | null {
+  const readonly = 'readonly' in node && node.readonly === true;
```

Comment:
There's no need to check this value for all the fields. This can be moved inline to only the node types that need it. As a benefit, `'readonly' in node` won't need to be checked for them.

Example:

```diff
-   const readonly = 'readonly' in node && node.readonly === true;
...
    case AST_NODE_TYPES.TSAbstractPropertyDefinition:
-      return readonly ? 'readonly-field' : 'field';
+      return node.readonly ? 'readonly-field' : 'field';
```

Reply:
Noted.